kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/utils/DeploymentUtils.java

```diff
@@ -235,6 +241,31 @@ public static void registryCredentialsSecret(String namespace) {
         }
     }
 
+    /**
+     * Collect cluster info.
+     *
+     * @param namespace the namespace
+     * @param testClassName the test class name
+     * @param testMethodName the test method name
+     */
+    public static void collectClusterInfo(String namespace, String testClassName, String testMethodName) {
+        String formattedDate = DateTimeFormatter.ofPattern("yyyy_MM_dd_HH_mm_ss")
+                .withZone(ZoneId.systemDefault())
+                .format(Instant.now());
+        List<String> executableCommand = List.of(cmdKubeClient(namespace).toString(), "cluster-info", "dump",
+                "--namespaces", String.join(",", namespace, Constants.KAFKA_DEFAULT_NAMESPACE),
+                "--output-directory", "/tmp/" + testClassName.replace(".", "_") + "_" +
```

Comment:
What's your idea for making the test artifact available to the github action / jenkins?

Reply:
Zip it and uploaded as an artifact in the jenkins run

Reply:
Sounds good.  You might want to consider passing an environment variable in as the location where to write the test artefact, defaulting to temp directory if not present.   That way, CI will know where to look for the .zip files.

Reply:
Yeah, I have it on my todo list